components/ContentFlow.brs

```diff
@@ -348,6 +348,8 @@ sub resumeVideoStream()
         m.videoPlayer.SetFocus(true)
         m.videoPlayer.control = "play"
         m.videoPlayer.seek = m.videoPositionAtAdBreakPause + m.streamSeekDuration
+        ' Add 1 second to avoid resuming too early, potentially due to rounding issues
+        m.videoPlayer.seek = m.videoPositionAtAdBreakPause + m.streamSeekDuration + 1 
```

Comment:
I think 1 second is the threshold for error here.... but I am going to try checking one more thing by manipulating the ad payload